types/setting.go

```diff
@@ -462,6 +465,16 @@ var (
 		Default:     "25",
 	}
 
+	SettingDefinitionStorageReservedPercentage = SettingDefinition{
+		DisplayName: "Storage Reserved Percentage",
+		Description: "The reserved percentage determines what percentage of disk space will not be used on a newly created Longhorn node.",
```

Comment:
Suggest `The reserved percentage specifies the percentage of disk space that will not be allocated to the default disk on each new Longhorn node.`

Reply:
Also, can we change the setting from `storage-reserved-percentage` to `storage-reserved-percentage-for-default-disk` instead?